tests/core_tests/wallet_tools.cpp

```diff
@@ -119,18 +132,12 @@ bool wallet_tools::fill_tx_sources(tools::wallet2 * wallet, std::vector<cryptono
 
     try {
       cryptonote::tx_source_entry src;
-      wallet_tools::gen_tx_src(mixin, cur_height, td, src, bt);
+      wallet_tools::gen_tx_src(mixin, cur_height, td, src, bt, fnc_in_accept);
 
       // Acceptor function
-      if (fnc_accept){
-        tx_source_info_crate_t c_info{.td=&td, .src=&src, .selected_idx=&selected_idx, .selected_kis=&selected_kis,
-            .ntrans=ntrans, .iters=iters, .sum=sum, .cur_utxo=cur_utxo};
-
-        bool take_it = (fnc_accept.get())(c_info, abort);
-        if (!take_it){
+      if (fnc_accept && !(fnc_accept.get())({.td=&td, .src=&src, .selected_idx=&selected_idx, .selected_kis=&selected_kis,
+                                              .ntrans=ntrans, .iters=iters, .sum=sum, .cur_utxo=cur_utxo}, abort))
```

Comment:
`abort` is a standard function defined in `cstdlib`, it's better to rename this variable.